app/controllers/check.controller.js

```diff
@@ -5,10 +5,37 @@
  * @module CheckController
  */
 
+const DetermineSupplementaryBillingFlagsService = require('../services/import/determine-supplementary-billing-flags.service.js')
 const ProcessLicenceReturnLogsService = require('../services/jobs/return-logs/process-licence-return-logs.service.js')
 
 const redirectStatusCode = 204
 
+/**
+ * A test end point for the licence supplementary billing flags process
+ *
+ * This endpoint takes a licenceId and an expired, lapsed and revoked date. It passes this onto the
+ * `DetermineSupplementaryBillingFlagsService` to test if it correctly flags the licence for supplementary billing. This
+ * normally happens during the licence import process.
+ *
+ * @param request - the hapi request object
+ * @param h - the hapi response object
+ *
+ * @returns {Promise<object>} - A promise that resolves to an HTTP response object with a 204 status code
+ */
+async function flagForBilling (request, h) {
+  const { licenceId, expiredDate, lapsedDate, revokedDate } = request.payload
+
+  const transformedLicence = {
+    expiredDate,
+    lapsedDate,
+    revokedDate
+  }
+
+  DetermineSupplementaryBillingFlagsService.go(transformedLicence, licenceId)
+
+  return h.response().code(redirectStatusCode)
```

Comment:
Not your fault as was already named this, but `204` is not a redirect status code. That should be `NO_CONTENT_STATUS_CODE` (because it is a constant as well as being incorrectly named!) https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204